Jenkinsfile

```diff
@@ -11,16 +12,18 @@ pipeline {
   }
 
   stages {
+
     stage('Run tests') {
       steps {
         sh './test.sh'
 
-        junit 'spec/reports/*.xml'
+        //junit 'spec/reports/*.xml'
```

Comment:
should this remain as a comment or be removed?

Reply:
comments removed